client/tests/e2e/specs/test_home_page.ts

```diff
@@ -26,16 +26,30 @@ describe('Check organization list', () => {
     cy.get('.organization-list-row__col').should('have.length', 5);
   });
 
+  it('Go to login page and enter wrong password', () => {
+    cy.contains('Boby McBobFace').click();
+    cy.get('.login-button-container > ion-button').should('have.class', 'button-disabled');
+    cy.get('#ms-password-input').find('input').invoke('attr', 'type').should('eq', 'password');
+    cy.get('#ms-password-input').find('input').type('Wr0ngP@ssw0rd.');
+    cy.get('.login-button-container > ion-button').should('not.have.class', 'button-disabled');
+    cy.get('.login-button-container > ion-button').click();
+    cy.get('@configPath').then(($elem) => {
+      const orgId = ($elem as unknown as string).split('/').slice(-1)[0];
+      cy.get('@consoleLog').should('have.been.calledWith', `Log in to ${orgId} with password 'Wr0ngP@ssw0rd.'`);
+    });
+    cy.contains('Could not login!');
```

Comment:
Should probably use a stricter condition: is the modal opened, is it danger-themed, ... I think `cy.contains` checks if it's in the DOM, not if it's visible to the user.
If it's going to be replaced by a toast then nevermind for now.

Reply:
it will be replaced yeah